### PR TITLE
Add reporting endpoint

### PR DESCRIPTION
**Pull request by jmiller656**
^

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def landing():
 @app.route('/salary',methods=['POST'])
 def salary():
 	data = get_data_from_request(request)
-	if data['amount'] and data['department'] and data['userID'] and data['name']:
```
Comment (jmiller656) [inline server.py]: I did this because we never put name in the database and we never use name

File: server.py [hunk 2]
```diff
@@ -46,13 +46,12 @@ def salary():
...
 			tag = data['tag']
 		except:
-			tag = "ssalary"
+			tag = "salary"
 		department = data['department']
 		if amount <= 0 or userId < 0 and not department in ACCEPTED_DEPARTMENTS:
 			return malformed_request()
 		taxAmount = amount * SALARY_TAX
 		amount += taxAmount
-		name = data['name']
```
Comment (jmiller656) [inline server.py]: Same for this

File: server.py [hunk 3]
```diff
@@ -143,6 +142,14 @@ def pay_tax():
...
 	pay_tax_amount(None,amount)
 	return ok_status()
+
+"""
+Internal API endpoint for getting data
+necessary for Reporting
+"""
+@app.route('/reporting',methods=['GET'])
+def reporting():
+	return get_reporting_info()
```
Comment (jmiller656) [inline server.py]: Tested this using python requests.
Received correct information by looking at response.content

File: server.py (hunk 1 above)
Comment (jmiller656) [inline server.py]: I would like to also remove the 'name' key from the API doc

Comment (btarr) [inline server.py]: will do

**Pull request merged**